packages/tpetra/core/src/Tpetra_MultiVector_def.hpp

```diff
@@ -908,8 +908,8 @@ namespace Tpetra {
     if (numSameIDs > 0) {
       const std::pair<size_t, size_t> rows (0, numSameIDs);
       if (copyOnHost) {
-        auto tgt_h = this->template getLocalView<HMS> ();
-        auto src_h = create_const_view (sourceMV.template getLocalView<HMS> ());
+        auto tgt_h = getDualView().view_host();
```

Comment:
It would be better to call `this->view_host()` directly.  In general, if we have the method on MultiVector, we should use the MultiVector method.  `Kokkos::DualView` is an implementation method, not the primary interface.